docs/config.md

```diff
@@ -1095,6 +1095,10 @@ key = "ssh-ed25519 AAAAC3NzaC1lZDI1NTE5AAAAIGj+J6N6SO+4P8dOZqfR1oiay2yxhhHnagH52
 sign-on-push = true
 ```
 
+### Manually signing commits
+
+You can use [`jj sign`](./cli-reference.md#jj-sign)/[`jj
+unsign`](./cli-reference.md#jj-unsign)to sign/unsign commits manually.
```

Comment:
@yuja, we can extend this section with an example for the workflow you mentioned in https://github.com/jj-vcs/jj/pull/4747#discussion_r1904854027, when making `-r` configurable.